support/util/imagemetadata.go

```diff
@@ -92,3 +94,18 @@ func ImageLabels(metadata *dockerv1client.DockerImageConfig) map[string]string {
 		return metadata.ContainerConfig.Labels
 	}
 }
+
+// GetPayloadImage get an image from the payload for a particular component
+func GetPayloadImage(ctx context.Context, hc *hyperv1.HostedCluster, component string, pullSecret []byte) (string, error) {
+	riprovider := &releaseinfo.RegistryClientProvider{}
```

Comment:
riprovider?

Reply:
I spelled it out more - releaseImageProvider.